platform-api/src/main/java/io/datacater/core/deployment/StaticConfig.java

```diff
@@ -112,6 +117,19 @@ private EnvironmentVariables() {}
         ConfigProvider.getConfig()
             .getOptionalValue("datacater.pythonrunner.image.containerPort", Integer.class)
             .orElse(50000);
+
+    static final String DEPLOYMENT_RESOURCES_REQUEST_MEMORY =
+        ConfigProvider.getConfig()
+            .getOptionalValue("datacater.deployment.resources.request.memory", String.class)
```

Comment:
Can you please make `request` plural, .i.e., `requests`, to match the `application.yml`?